### ISSUE TITLE
Is there any problem when update to version 4.0.2 from 3.4.5？

### ISSUE THREAD

**Issue by KerwinAusten** [brettwooldridge/HikariCP#1737]
version 4.0.0 added a very useful property - keepaliveTime. But spring-boot-2.3.8 used hikariCp 3.4.5. Could I update the version directly?

**Comment by bwertman-rc** [comment]
I also have this question. Documentation about upgrading from 3.X to 4.0 would be helpful. If there were no breaking changes, why did the major version change?

**Comment by dev-sg-08** [comment]
look here: [https://github.com/brettwooldridge/HikariCP/blob/dev/CHANGES](https://github.com/brettwooldridge/HikariCP/blob/dev/CHANGES)

**Comment by KerwinAusten** [comment]
> look here: https://github.com/brettwooldridge/HikariCP/blob/dev/CHANGES

Yeah, I have seen that document, but these changes look like a tiny update.

What confused me is why the major version updated to 4.0?😕
